src/flow_solver/flow_solver_factory.cpp

```diff
@@ -46,7 +46,7 @@ ::select_flow_case(const Parameters::AllParameters *const parameters_input,
             std::shared_ptr<FlowSolverCaseBase<dim, nstate>> flow_solver_case = std::make_shared<NACA0012<dim,nstate>>(parameters_input);
             return std::make_unique<FlowSolver<dim,nstate>>(parameters_input, flow_solver_case, parameter_handler_input);
         }
-    } else if (flow_type == FlowCaseEnum::advection_periodic){
+    } else if ((flow_type == FlowCaseEnum::advection_periodic)||(flow_type == FlowCaseEnum::burgers_periodic)){
```

Comment:
if the only difference between these is the `pde_type` you specify in the `.prm` file, consider colapsing these two flow types into `periodic_1d_unsteady`

Reply:
This is a good suggestion, I have changed them to both be `periodic_1D_unsteady` and any differences use the `pde_type` as an indicator.